src/pages/reference/compare-to-heroku.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Railway vs. Heroku
+---
+
+## Summary
+
+Railway is a simple and powerful deployment platform that focuses on giving you and your team a deployment plane that radically increases developer efficiency.
+
+We are alike in the following ways:
+
+- GitHub repo deployments
+- CLI tooling
+- Built-in databases
+
+We differ in the following:
+
+- We have an outsized focus on support and developer experience. A highly engaged community and the Railway team stand at the ready to help you scale
+- Resource-based pricing, only pay for what you use.
+- Better developer experience. We offer PR deploys, variable management, rapid builds, and local development flows.
+- And much more...
+
+## Background
+
+We humbly know that users have a lot of options when it comes to deploying their workloads.
+
+One prevalent option is Heroku. Heroku became known for one of the easiest ways to deploy code, thanks to opinionated CLI tooling, first-class Ruby support, and built-in integrations such as databases and Salesforce.
+
+Railway is a natural replacement for people looking to move on from Heroku or looking for similar alternatives.
+
+However, one of the most common questions we receive is: How are we different or better than Heroku?
+We owe a lot to the pioneers that were the founders of Heroku. However, in 2007, Heroku's founding, the state of cloud deployments was much different than what they were today. As a result, their product reflects the context of the state of the cloud was then.
+
+## Differences
+
+### Builds
+
+Part of the limitations faced by the Heroku team during the early 2010s was the question of how to best ship code.
+
+At the time, the idea of unikernels and Docker didn't take hold until much later. As a result: their build solution, Buildpacks, used mostly complex Bash scripts to introspect into code to build your app.
+
+In the current era of software development, we feel that in some respects, knowing Docker to know how to ship your app is too much boilerplate for the average developer. But, we still want to offer the developer all the benefits of reproducibility, speed, and ease of use when deploying images, hence why we built our builder called Nixpacks.
+It's open source (https://github.com/railwayapp/nixpacks), and it's also swift.
+We detect more languages, builds are faster with the help of intelligent caching, and lastly, builds are reproducible. (A common frustration of Buildpacks.)
+Those moving from Heroku to Railway will find that we support CNCF Buildpacks just like Heroku does if you want to keep using their build solution.
```

Comment:
Heroku doesn't use CNF buildpacks. They have their own version that are slightly different

```suggestion
Those moving from Heroku to Railway will find that we support the Heroku CNCF Buildpacks.
```